CMakeLists.txt

```diff
@@ -727,6 +728,32 @@ if (NOT Build_Protobuf)
     NAMESPACE ONNX::
   )
 endif()
+
+if(ONNX_USE_UNITY_BUILD)
```

Comment:
This section does not seem to add any new target. I assume it is meant to be used when onnx is compiled as a dependency. Is it possible to know more about it? Some files are removed, are the not needed?

Reply:
I added comments. Tell me if it is clear enough.